langfilter/langfilter.php

```diff
@@ -55,9 +55,9 @@ function langfilter_addon_settings(App $a, &$s)
 	$t = get_markup_template("settings.tpl", "addon/langfilter/");
 	$s .= replace_macros($t, [
 		'$title'         => L10n::t("Language Filter"),
-		'$intro'         => L10n::t('This addon tries to identify the language of a postings. If it does not match any language spoken by you (see below) the posting will be collapsed. Remember detecting the language is not perfect, especially with short postings.'),
+		'$intro'         => L10n::t('This addon tries to identify the language posts are writen in. If it does not match any language specifed below, posts will be hidden by collapsing them. (Note: detecting languages is not perfect, especially for short texts.)'),
```

Comment:
"This addon guesses the language posts are written in. Their content will be collapsed if it doesn't match any language specified below. Note: Language detection is unreliable for short content (<200 characters)"

Reply:
So true!

Reply:
This begs the question whether it should not be set to 200 characters in the default settings.

It would like to dispense with this sentence here in the explanation and add this below where one adjusts this setting.

Reply:
200 is a really safe margin but it lets pass many shorter posts as well.

For reference I'm using minimum confidence: 23% and minimum text length: 100 characters, but BBCodes are often foiling the character count.

Okay to move the note in the addon overview.

Reply:
Cool! 

I have similar settings. Minimum confidence: 25% and minimum text length: 80 characters